scripts/lib/sphinx/unifiedParser.ts

```diff
@@ -10,7 +10,82 @@
 // copyright notice, and modified files need to carry a notice indicating
 // that they have been altered from the originals.
 
+import { visit } from "unist-util-visit";
+import { Root } from "mdast";
+import remarkMath from "remark-math";
+import remarkMdx from "remark-mdx";
+import { unified } from "unified";
+import rehypeParse from "rehype-parse";
+import rehypeRemark from "rehype-remark";
+import remarkStringify from "remark-stringify";
+import remarkGfm from "remark-gfm";
+import { last, first, initial, tail } from "lodash";
+import { Handle } from "hast-util-to-mdast";
+import { removePrefix, removeSuffix } from "../stringUtils";
+
 export const remarkStringifyOptions = {
   fences: true,
   resourceLink: true,
 };
+
+export async function sphinxHtmlToMarkdownUnifiedPlugin(
+  htmlToProcess: string,
+  handlers?: Record<string, Handle>,
```

Comment:
Let me know your opinion about moving the unified plugin to this file. I'm not sure it's the best option because to end up with the same result, we need all the transformations done in the `sphinxHtmlToMarkdown.ts`. The only advantage I see is to reduce the amount of code we have in that script, but all our unified plugins are very different from each other to make something general.

Even though I'm more on board with not moving it, I wanted to show you the change in this PR to know your opinion.

Reply:
Yeah I wouldn't move it until we in the same PR switch other code to use the same config.

We might still want to split out sphinxHtmlToMarkdown.ts into several smaller follows. But that would be splitting out more than only this code.

Reply:
I moved it back to the `sphinxHtmlToMarkdown.ts` file!